Tests/SwiftStdlibTests/ArrayExtensionsTests.swift

```diff
@@ -145,7 +145,8 @@ class ArrayExtensionsTests: XCTestCase {
     }
     
     func testDuplicatesRemoved() {
-        XCTAssertEqual([1, 1, 2, 2, 3, 3, 3, 4, 5].duplicatesRemoved(), [1, 2, 3, 4, 5])
+        XCTAssertEqual([1, 2, 2, 3, 4, 5, 5].duplicatesRemoved(), [2, 5])
```

Comment:
See how the test has different behavior here

Reply:
@SD10 Sorry, i saw this 
```swift
/// SwifterSwift: Return array with all duplicate elements removed.
///
///     [1, 1, 2, 2, 3, 3, 3, 4, 5].duplicatesRemoved() -> [1, 2, 3, 4, 5])
///     ["h", "e", "l", "l", "o"].duplicatesRemoved() -> ["h", "e", "l", "o"])
///
/// - Returns: an array of unique elements.
///
```
 on the docs exemples and thougth the implementation was wrong, but notice that actually the exemples are inconsistent as soon as created the PR, already fixed :)